Models/DTO/Administrator/FilterPharmacyReportDTO.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.ComponentModel.DataAnnotations;
+
+namespace API.Models.DTO.Administrator
+{
+    public class FilterPharmacyReportDTO
+    {
+        [Required]
+        [JsonProperty("id")]
+        public int Id { get; set; }
+
+        [Required]
+        [DataType(DataType.DateTime)]
+        [DisplayFormat(DataFormatString = "{0:yyyy-MM-dd}", ApplyFormatInEditMode = true)]
+        [JsonProperty("datefrom")]
```

Comment:
Maybe it would be more readable to serialize as `date-from`?

Reply:
We can use only letters without special symbols.